.circleci/config.yml

```diff
@@ -137,7 +137,7 @@ jobs:
       BOOTSTRAP_URL: "https://boyar-testnet-bootstrap.s3-us-west-2.amazonaws.com/boyar/config.json"
       BOOTSTRAP_S3_URI: "s3://boyar-testnet-bootstrap/boyar/config.json"
       TESTNET_NODE_IP: "35.161.123.97"
-      TESTNET_IGNORE_IPS: "54.149.67.22"
+      TESTNET_IGNORE_IPS: ""
```

Comment:
what's this?

Reply:
This IP is not relevant  - It was there because @netoneko  was testing something special on that node and it was not available for me. So I made an option to exclude certain nodes from the network so that my health check post deployment will work